fuzzers/072-ordered_wires/run_fuzzer.py

```diff
@@ -3,13 +3,15 @@
 import sys
 import subprocess
 import signal
-from multiprocessing import Pool
+from multiprocessing import Pool, Lock
 from itertools import chain
 import argparse
 
 # Can be used to redirect vivado tons of output
 # stdout=DEVNULL in subprocess.check_call
 
+MP_LOCK = Lock()
```

Comment:
Probably better to pass the lock as part of the arglist than to use a global.

Reply:
So, the lock cannot be zipped  with the other arguments. Instead, I have kept this variable global, but I have encapsulated it in an init function that is passed at the creation time of the Pool of workers.